src/controller/record.ts

```diff
@@ -1,6 +1,9 @@
 import express from "express";
 import { getConnection } from "typeorm";
 
+import * as Cord from '@cord.network/sdk';
+import * as VCUtils from '@cord.network/vc-export';
```

Comment:
nitpick: as the practise, these cord imports were inside of `src/cord/` directory. Good to follow the same here too. 

Not a blocker for merge, but a note to leave for future developer to pickup and fix.